src/lib/graphs.js

```diff
@@ -439,7 +440,13 @@ function update(timeInSecs, augmint) {
         }
 
         // redraw:
-        graph.chart.update();
+        // every 50 days
+        if(rates.accelerate && !(rates.rend % rates.refreshPeriod) || rates.rend == (rates.length - 1)){
+            graph.chart.update();
+        }
+        if(!rates.accelerate) {
```

Comment:
I'm sure you can write this as one if ;)
but I would rather remove this logic from here so that graph doesn't need to know about his param.
I.e. split current update fx to an updateData and a refreshGraph fx. ui.js'  render() should call updateData() (just as now) and call refreshGraph it's required. 
So that graph doesn't need to know about sim params